Scripts/signin.js

```diff
@@ -1,48 +1,57 @@
-import {inputFocus, inputBlur, printError, errorType} from './signFunction.js'
-import {loginCheck} from './accountData.js';
-import passToggleReset from './passwordToggle.js';
+import {focusToInput, blurFromInput, errPrint, isThisLoginWasSuccessful, ERROR_TYPE} from './signFunction.js'
+import pwTypeToggleBtnAssign from './pwTypeToggle.js';
+import {pageRedirectToFolderIfClientHasToken} from './tokenHandle.js'
+
+// 토큰이 있을 경우 folder로 리다이렉트
+pageRedirectToFolderIfClientHasToken();
 
 // 비밀번호 토글 초기화 및 함수 등록
-passToggleReset();
+pwTypeToggleBtnAssign();
 
-const loginEmailInput = document.querySelector('#loginEmailInput');
-const loginPasswordInput = document.querySelector('#loginPasswordInput');
+const emailInput = document.querySelector('#emailInput');
+const pwInput = document.querySelector('#pwInput');
 const confirmBtn = document.querySelector('#confirmBtn');
 
 // 에러 메시지 출력 장소
-const emailErrorSection = loginEmailInput.parentElement.lastElementChild;
-const passwordErrorSection = loginPasswordInput.parentElement.parentElement.lastElementChild;
+const emailErrTextSection = document.querySelector('#emailErr')
+const pwErrTextSection = document.querySelector('#pwErr');
 
 
 // 이메일 리스너
-loginEmailInput.addEventListener('input' , () => inputFocus(emailErrorSection));
-loginEmailInput.addEventListener('focus' , () => inputFocus(emailErrorSection));
-loginEmailInput.addEventListener('blur' , () => inputBlur(errorType.email, emailErrorSection));
+emailInput.addEventListener('input' , (e) => focusToInput(e, emailErrTextSection));
+emailInput.addEventListener('focus' , (e) => focusToInput(e, emailErrTextSection));
+emailInput.addEventListener('blur' , (e) => blurFromInput(e, emailErrTextSection, ERROR_TYPE.EMAIL_SECTION_BLANK));
 
 
 // 패스워드 리스너
-loginPasswordInput.addEventListener('input' , () => inputFocus(passwordErrorSection));
-loginPasswordInput.addEventListener('focus' , () => inputFocus(passwordErrorSection));
-loginPasswordInput.addEventListener('blur' , () => inputBlur(errorType.password, passwordErrorSection));
+pwInput.addEventListener('input' , (e) => focusToInput(e, pwErrTextSection));
+pwInput.addEventListener('focus' , (e) => focusToInput(e, pwErrTextSection));
+pwInput.addEventListener('blur' , (e) => blurFromInput(e, pwErrTextSection, ERROR_TYPE.PW_SECTION_BLANK));
 
 
 // 로그인 정보 제출
-const submitLoginData = function (event)  {
-   event.preventDefault();
+const signinDataSubmit = async function (e)  {
+   e.preventDefault();
    
    //입력받은 값
-   const emailValue = loginEmailInput.value;
-   const passwordValue = loginPasswordInput.value;
-   
-   // 계정정보 유효 확인
-   const isAccountSigned = loginCheck(emailValue, passwordValue);
+   const emailValue = emailInput.value;
+   const passwordValue = pwInput.value;
+   const triedLoginAccountData = {
+      email : emailValue,
+      password : passwordValue
+   }
```

Comment:
email 값과 password 값이 재사용되지 않으면 아래처럼 하셔도 될 것 같아요.

```suggestion
   const triedLoginAccountData = {
      email : emailInput.value,
      password : pwInput.value
   }
```

Reply:
코드를 다시 확인해보고 고쳐보겠습니다. 감사합니다!